server/controllers/auctions.js

```diff
@@ -36,9 +36,9 @@ router.post('/', authenticate, (req, res) => {
     res.status(500).send(serverErr);
   });
 });
-// SEEMS LIKE NO ONE IS POSTING TO THIS ENDPOINT
-router.post('/ongoing', (req, res) => {
-  const { user } = req.body;
+
+router.get('/ongoing', authenticate, (req, res) => {
```

Comment:
because they are the auctions that a individual user is bidding on, has won, etc.